ingest/sqs/domain/mocks/sqs_ingester_mock.go

```diff
@@ -43,3 +44,7 @@ func (s *SQSIngesterMock) ProcessChangedBlockData(ctx types.Context, changedPool
 	s.LastChangedPoolsObserved = changedPools
 	return s.ChangedBlockDataError
 }
+
+func (SQSIngesterMock) PublishBlock(ctx types.Context, block indexerdomain.Block) error {
+	return nil
+}
```

Comment:
Let's remove this. It's defined on the ingester now AFAIU

Reply:
Done